pyclesperanto_prototype/_tier0/_available_device_names.py

```diff
@@ -0,0 +1,13 @@
+from .._tier0._pycl import filter_devices
+from typing import List
+
+def available_device_names() -> List[str]:
```

Comment:
maybe pass `**kwargs` to filter_devices (and indicate that it is being done in the docs) so that someone could say `available_device_names(dev_type="gpu")` for instance